openverse_catalog/dags/common/loader/smithsonian_unit_codes.py

```diff
@@ -61,74 +28,34 @@ def get_new_and_outdated_unit_codes(unit_code_set, sub_prov_dict=SUB_PROVIDERS):
     new_unit_codes = unit_code_set - sub_provider_unit_code_set
     outdated_unit_codes = sub_provider_unit_code_set - unit_code_set
 
-    if bool(new_unit_codes):
-        logger.info(
-            f"The new unit codes {new_unit_codes} must be added to "
-            f"the SMITHSONIAN_SUB_PROVIDERS dictionary"
-        )
-
-    if bool(outdated_unit_codes):
-        logger.info(
-            f"The outdated unit codes {outdated_unit_codes} must be "
-            f"deleted from the SMITHSONIAN_SUB_PROVIDERS dictionary"
-        )
-
     return new_unit_codes, outdated_unit_codes
 
 
 def alert_unit_codes_from_api(
-    postgres_conn_id,
-    unit_code_table="smithsonian_new_unit_codes",
     units_endpoint=UNITS_ENDPOINT,
     query_params=PARAMS,
 ):
+    """
+    Alert in Slack if human intervention is needed to update the
+    SMITHSONIAN_SUB_PROVIDERS dictionary
+    """
     response = requests.get(units_endpoint, params=query_params)
     unit_code_set = set(response.json().get("response", {}).get("terms", []))
     new_unit_codes, outdated_unit_codes = get_new_and_outdated_unit_codes(unit_code_set)
 
-    initialise_unit_code_table(postgres_conn_id, unit_code_table)
-
-    postgres = PostgresHook(postgres_conn_id=postgres_conn_id)
+    if bool(new_unit_codes) or bool(outdated_unit_codes):
+        message = "\n*Updates needed to the SMITHSONIAN_SUB_PROVIDERS dictionary*:\n\n"
 
-    """
-    Populate the table with new unit codes
-    """
-    for new_unit_code in new_unit_codes:
-        postgres.run(
-            dedent(
-                f"""
-                INSERT INTO public.{unit_code_table}
-                (new_unit_code, action)
-                VALUES (
-                  '{new_unit_code}', 'add'
-                );
-                """
-            )
-        )
+        if bool(new_unit_codes):
+            codes_string = "\n".join(f"  - `{code}`" for code in new_unit_codes)
+            message += "New unit codes must be added:\n"
+            message += codes_string
+            message += "\n"
```

Comment:
Might be worth doing a docstring here:
```suggestion
            message += f"""\
New unit codes must be added:
{codes_string}
"""
```

Reply:
We could do that! I was annoyed by what the lack of whitespace does to the formatting, but obviously not a big deal 😄

Reply:
I hear it :sweat_smile: We could use [`textwrap.dedent`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) as well, but sometimes that can misbehave if you're formatting in a variable that includes newlines (because those lines don't have the same number of starting spaces).